computable/tests/sign_test.py

```diff
@@ -22,13 +22,25 @@ def test_send_some_eth(w3, user):
     new_user_bal = w3.eth.getBalance(user)
     assert new_user_bal == Web3.toWei(1, 'finney')
 
-def test_send_some_ether_token(w3, user, ether_token):
+def test_send_some_ether_token(w3, pk, user, ether_token):
```

Comment:
The tests previously used the initial balance which no longer exists for `EtherToken`.